upup/models/cloudup/resources/addons/digitalocean-cloud-controller.addons.k8s.io/k8s-1.8.yaml.template

```diff
@@ -47,11 +47,10 @@ spec:
           operator: Exists
           tolerationSeconds: 300
       containers:
-      - image: digitalocean/digitalocean-cloud-controller-manager:v0.1.7
+      - image: digitalocean/digitalocean-cloud-controller-manager:v0.1.15
         name: digitalocean-cloud-controller-manager
         command:
           - "/bin/digitalocean-cloud-controller-manager"
-          - "--cloud-provider=digitalocean"
```

Comment:
Maintainer of DO's CCM here.

This flag is safe for removal as we now set it internally in the CCM code.

Reply:
👍